specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/gallery.json

```diff
@@ -3144,6 +3197,30 @@
           "description": "region of the Gallery Image Version."
         }
       }
+    },
+    "latestVersion": {
+      "readOnly": true,
+      "description": "The name of the latest version in the region.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id"
```

Comment:
Make these descriptions more specific. ex: 'Resource Id of the latest version of a specific image.' (if that description is correct). Same for the other 2 properties here.